### PR TITLE
Demographics ft metadata

### PR DESCRIPTION
**Pull request by VladDubrovskis**

### CODE REVIEW

File: src/layouts/_archive/layout_one/layout.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-		"pageType" : "ind",
-		"slots" :require("../front-page/banners_between_everything")
+	'front-page' : {
+		'pageType' : 'ind',
+		'slots' :require('../front-page/banners_between_everything')
 	},
-	"grumman" : {
-		"pageType" : "art",
-		"slots" : require('../grumman/right_rail_and_in_article')
+	'grumman' : {
```
Comment (commuterjoy) [inline src/layouts/_archive/layout_one/layout.js]: now called article

File: src/layouts/_archive/layout_one/layout.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+		'pageType' : 'art',
+		'slots' : require('../grumman/right_rail_and_in_article')
 	},
-	"dobi" : {
-		"pageType" : "str",
-		"slots" : require('../dobi/in_between_stream')
+	'dobi' : {
```
Comment (commuterjoy) [inline src/layouts/_archive/layout_one/layout.js]: now called stream

File: src/layouts/_archive/layout_one/layout.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+		'pageType' : 'str',
+		'slots' : require('../dobi/in_between_stream')
 	},
-	"iluyshin" : {
-		"pageType" : "ind",
-		"slots" : []
+	'iluyshin' : {
```
Comment (commuterjoy) [inline src/layouts/_archive/layout_one/layout.js]: now called myft

**Comment by commuterjoy** [comment]
:-1: 

This seems to include ft-metadata which looks like a vague way to scrape data out of cookies (which we are trying to get rid of) and global `FT` objects (which don't exist on Next).

/cc @matthew-andrews

**Comment by commuterjoy** [comment]
I've said a few times - though possibly not to you @VladDubrovskis - that https://github.com/Financial-Times/next-ijento is a much simpler architecture to keep the client-side code clean, and allow us to not pick up dependencies on old cookies and gradually/transparently migrate to new APIs.

**Comment by VladDubrovskis** [comment]
I think the whole idea of this PR and the ft-metadata is exactly that, to take the code that is relying on cookies and FT and move out to a separate module that can be phased out eventually. 

The .page() call that is relying on the global "FT" variable is only used on FT.com and not Next, the .user() call is however used in both places, thus the reason for moving code out to a small outside module. 

This is so we can get demographics data. All of the code is existing code that was just moved rather than added.

**Comment by commuterjoy** [comment]
@VladDubrovskis @robin-marr-ft what happened to the plan to pull this demographic data out of an API?

**Comment by VladDubrovskis** [comment]
Needs more work

**Pull request closed without merging**